source/refined-github.css

```diff
@@ -191,3 +191,85 @@ div[data-target='readme-toc.content'] div.blob-wrapper img {
 [data-selected-links^='repo_milestones ']:has([title='0']) svg {
 	margin-right: 20px; /* Push the text farther away from the text so it's cropped out */
 }
+
+/* Heatmap colors copied from GH, because GH does not load them on every page #5637 */
```

Comment:
These should be in their own `heat-map.css` file, with a comment that includes the source of this code so it can be updated again in the future.

Reply:
Updated in latest. I put the file in the root for now since it can be used across many features.